README.md

```diff
@@ -117,22 +117,26 @@ The command should return the absolute path to the binary if it's found in the s
 
 How to configure Neovim with lua modules: https://neovim.io/doc/user/lua-guide.html#lua-guide-config
 
-Use lspconfig to create a custom contextive language server configuration and initialize the language server by calling the setup function. The following lua snippet needs to be included in the `init.lua` file either directly or from another lua module like `lspconfigs.lua`.
+Contextive Language Server is also provided with [Mason](https://github.com/williamboman/mason.nvim).
```

Comment:
Now that mason can download the language server - would it make sense to move this to the 'all in one' section above?

Also, what is the mason process for initiating the download of the server - is there a `mason install contextive` type cli command, or is it an interactive process inside neovim? 

Apologies if these are all obvious to neovim users, I'm a super neovim newbie!

Reply:
When I think of _All in one_, I imagine everything bundled together in a single _box_, like a standalone plugin. However, in this case, we need to configure and install separate components.

I've included a command in the document that you can use to install Contextive with Mason!

Mason offers at least two methods to install the language servers: an interactive UI and a direct command call.